_posts/platform/deployment/2000-01-01-deploy-with-git.md

```diff
@@ -60,3 +60,28 @@ issues]({% post_url platform/getting-started/2000-01-01-troubleshooting-ssh %}).
 See [Deployment Environment]({% post_url platform/app/2000-01-01-environment
 %}#build-environment) to configure your application and [Procfile]({% post_url
 platform/app/2000-01-01-procfile %}).
+
+### Clone your code
```

Comment:
Please titleize the titles. I will send you by MP a link to our internal doc about that